test/helpers/index.js

```diff
@@ -0,0 +1,11 @@
+module.exports = { isAround, sleep }
+
+function isAround (actual, expected, precision = 5) {
+  const diff = Math.abs(actual - expected)
```

Comment:
5 is prop too little for the macvm on ci (they are very slow).
Also timers guarantee that it never fires too early, ie actualDelay >= expectedDelay always

Reply:
True, Mac CI just failed, there is absolutely no way to test a rough precision?

Reply:
No testing timers suck on ci, due to slowness. maybe make the precison depending on process.platform also